docs/applications/big-data/big-data-in-the-linode-cloud-streaming-data-processing-with-apache-storm.md

```diff
@@ -0,0 +1,1095 @@
+
+## Start a New Topology
+
+If you or a developer have created a topology, perform these steps to start a new topology on one of your Linode Storm clusters:
+
+{: .note}
+> The developer should have `clusteradmin` (or `clusterroot`) authorization to log in to the client node of the target Storm cluster.
+>
+> Optionally, to get the IP address of client node, the developer should have `clustermgrguest` (or `clustermgrroot`) authorization to log in to the Cluster Manager Linode. If the IP address is known by other methods, this authorization is not required.
+
+1.  Package your topology along with all the third party classes on which they depend into a single JAR (Java Archive) file. 
+
+2.  If multiple clusters are deployed, select the target Storm cluster to run the topology on. Get the public IP address of the client node of the target cluster. See [cluster description](#describe-a-storm-cluster) for details on how to do this.
+
+3.  Transfer the topology JAR from your local workstation to client node:
+
+        scp -i ~/.ssh/private-key local-topology-path clusteradmin@public-ip-of-client-node:topology-jar
+
+    Substitute `private-key` for the private key of the Storm client, `local-topology-path` for the local filepath of the JAR file, `PUBLIC-IP-OF-CLIENT-NODE` for the IP address of the Storm client, and `topology-jar` for the filepath you'd like to use to store the topology on the client node.
+
+4.  Log in to the client node as `clusteradmin`, substituting the appropriate values:
+
+        ssh -i ~/.ssh/private-key clusteradmin@PUBLIC-IP-OF-CLIENT-NODE
+
+5.  Submit the topology to the cluster:
+
+        cd /opt/apache-storm-0.9.5/bin
+        ./storm jar topology-jar.jar main-class arguments-for-topology
+
+    Substitute `topology-jar.jar` for the path of the JAR file you wish to submit, `main-class` with the main class of the topology, and `arguments-for-topology` for the arguments accepted by the topology's main class.
+
+6.  [Monitor the execution of the new topology.](#monitor-a-storm-cluster-using-storm-ui)
+
+{: .note}
+> The Storm UI will show only information on the topology's execution, not the actual data it is processing. The data, including its output destination, is handled in the topology's JAR files. 
+
+## Other Storm Cluster Operations
+
+In this section, we'll cover additional operations to manage your Storm cluster once it's up and running.
+
+All commands in this section should be performed from the `storm-linode` directory on the cluster manager Linode. You will need `clustermgr` privileges unless otherwise specified.
+
+### Expand a Storm Cluster
+
+If the supervisor nodes of a St
```

Comment:
I think we should leave the example in, but explain as you just did that source and target are reverse from normal `cp`, and maybe include the syntax example you noted. Since I've already give you a PR, I'll leave that to you to add after you merge it. I *hate* dealing with merge conflicts.